src/modules/acc/getters.js

```diff
@@ -1,10 +1,20 @@
 /* eslint no-shadow: ["error", { "allow": ["getters"] }] */
 import { key, PrivateKey } from 'bitsharesjs';
+import config from '../../../config'
+
+console.log(config);
 
 const parseOpenOrders = (orders, rootGetters) => {
   const parsedActiveOrders = orders.map(order => {
     // need to get if from somewhere
-    const isBid = true;
+    let isBid;
+    const { symbol: baseSymbolInSellPrice } = rootGetters['assets/getAssetById'](order.sell_price.base.asset_id);
+    if (config.marketBases.some(x => x === baseSymbolInSellPrice)) {
```

Comment:
this is the same as
`return config.marketBases.some(baseSymbol => baseSymbol === baseSymbolInSellPrice))`